pkg/operator/v1helpers/helpers.go

```diff
@@ -259,3 +261,19 @@ func (agg aggregate) Error() string {
 func (agg aggregate) Errors() []error {
 	return []error(agg)
 }
+
+// MapToEnvVars converts a string-string map to a slice of corev1.EnvVar-s
+func MapToEnvVars(mapEnvVars map[string]string) []corev1.EnvVar {
+	if mapEnvVars == nil {
+		return []corev1.EnvVar{}
```

Comment:
why not nil?

Reply:
@mfojtik convinced me it'll be easier to use this in `append`

Reply:
it is not. This has smell. Don't do magic.